Content.Server/Players/PlayTimeTracking/PlayTimeTrackingSystem.cs

```diff
@@ -74,19 +74,28 @@ private bool IsBypassingChecks(ICommonSession player)
 
     private void CalcTrackers(ICommonSession player, HashSet<string> trackers)
     {
-        if (_afk.IsAfk(player))
+        //SS220-aghost-playtime begin
+        // Checking for attached entity to prevent tracking for players in lobby
+        if (_afk.IsAfk(player) || player.AttachedEntity == null)
             return;
 
         if (_adminManager.IsAdmin(player, includeDeAdmin: false))
         {
-            trackers.Add(AdminTrackerTime);
+            trackers.Add(PlayTimeTrackingShared.TrackerAdmin);
+            if (player.AttachedEntity is { } attachedEntity &&
```

Comment:
```suggestion
            if (player.AttachedEntity is { Valid: true } attachedEntity &&
```
На всякий случай, потому что у удаленных (невалидных) энтити нет метадаты